assets/js/admin/orders.js

```diff
@@ -11,4 +11,91 @@ jQuery( document ).ready( ( $ ) => {
 
 	'use strict';
 
+	const isCommerceOrder = Boolean( wc_facebook_commerce_orders.is_commerce_order );
+
+	window.WCFacebookCommerceOrderOperations = {
```

Comment:
I don't think we need to create a window object for these methods. We usually don't unless we plan on instantiating it, and when we do it's usually described on the discovery doc.

Reply:
Understood, so you recommend removing it from the window scope?